dist/rpm.dockerfile

```diff
@@ -35,27 +35,20 @@ case "${OS_VERSION}" in
         chmod a+x /etc/scl_enable
         ;;
     8|9)
-        yum install -y epel-release rubygems ruby-devel
-        ;;
-    *)
-        if [ "${OS_NAME}" = "fedora" ]; then
-            yum install -y rubygems ruby-devel
-        else
-            yum install -y epel-release rubygems ruby-devel
-        fi
+        yum install -y rubygems ruby-devel glibc-langpack-en glibc-locale-source
         ;;
 esac
 
 # Disable postgres
-if command -v dnf; then dnf -qy module disable postgresql; fi
+#if command -v dnf; then dnf -qy module disable postgresql; fi
+
+EOF
 
+RUN <<EOF
 # System dependencies
 yum install -y \
     gettext \
     rpm-build \
-    rpm-devel \
-    rpmlint \
```

Comment:
don't we need rpm lint if we are lifting aka this [line](https://github.com/timescale/promscale_extension/blob/develop/tools/package#L296) of tools/package?

Reply:
I've reverted this.